### PR TITLE
Location gem tracking

### PR DESCRIPTION
**Pull request by msmollin**
Next step of #31

### CODE REVIEW

File: ios-sdk.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -237,9 +241,20 @@
 			path = fonts;
 			sourceTree = "<group>";
 		};
+		DB366BDC1DF8BE090045C15F /* Images */ = {
+			isa = PBXGroup;
+			children = (
+				DB366BDD1DF8BE090045C15F /* ic_find_me_normal@2x.png */,
```
Comment (karimnaaji) [inline ios-sdk.xcodeproj/project.pbxproj]: What happens on 1x devices?

Comment (msmollin) [inline ios-sdk.xcodeproj/project.pbxproj]: They don't show the button. I actually realized this after I submitted it, but noticed I didn't have a large enough button asset for it, so @souperneon hooked me up - https://github.com/mapzen/ios/issues/104

I'll be updating this PR in a few with the new assets.

File: ios-sdk/src/MapViewController.swift [hunk 2]
```diff
@@ -61,6 +93,11 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate {
...
             markerSetVisible(marker, visible: true)
         }
+
+        if (shouldFollowCurrentLocation) {
+            print("Updating for current lat: \(location.coordinate.latitude) & long: \(location.coordinate.longitude)")
+            centerOnCurrentLocation()
+        }
     }
 
     public func authorizationDidSucceed() {
```
Comment (karimnaaji) [inline ios-sdk/src/MapViewController.swift]: Ok I see, requesting a location might be good, so as soon as you give the authorization the user see his current location from closer zoom level.

File: ios-sdk.xcodeproj/project.pbxproj (hunk 1 above)
Comment (karimnaaji) [inline ios-sdk.xcodeproj/project.pbxproj]: 👍

**Pull request merged**